newIDE/app/src/AssetStore/AssetDetails.js

```diff
@@ -64,6 +65,7 @@ const styles = {
     maxHeight: '100%',
     verticalAlign: 'middle',
     pointerEvents: 'none',
+    flex: 0.6,
```

Comment:
Where is this coming from?

Reply:
This is en empirical value. The parent is 250x300, the image will be this size at maximum.
For smaller images, if this is pixel art, I'm not afraid to display them at 100% but for small smoothed images, the rendering is not that great, so I preferred a smaller ratio, that is a compromise between rendering and view that is more zoomed than in the search results